### PR TITLE
EqualsExlcude notation, HashCodeExclude notation and ToStringExludeNotat...

### PR DESCRIPTION
**Pull request by FelipeAdorno**
New pullrequest for fixing all code reviews.

### CODE REVIEW

**Comment by britter** [comment]
Very nice, thank you!

File: src/main/java/org/apache/commons/lang3/builder/EqualsExclude.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+package org.apache.commons.lang3.builder;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * Use this annotation to builds a equals excluding the annotated field.
+ * @author Felipe Adorno (felipeadsc@gmail.com)
```
Comment (britter) [inline src/main/java/org/apache/commons/lang3/builder/EqualsExclude.java]: We don't use the author tag, but instead attribute contributions in the contributors section of [pom.xml](pom.xml). Would it be okay for you to remove the author tags?

**Comment by FelipeAdorno** [comment]
sorry @britter, I remove author tag.

**Comment by britter** [comment]
Nice work @FelipeAdorno! Kudos for fighting through the review process.

**Pull request merged**